sdk/appconfiguration/app-configuration/review/app-configuration.api.md

```diff
@@ -23,35 +25,56 @@ export type AddConfigurationSettingParam<T extends string | FeatureFlagValue | S
 export interface AddConfigurationSettingResponse extends ConfigurationSetting, SyncTokenHeaderField, HttpResponseField<SyncTokenHeaderField> {
 }
 
+// @public
+export type AppConfigurationApiVersion = (typeof AppConfigurationApiVersion)[keyof typeof AppConfigurationApiVersion];
+
+// @public
+export const AppConfigurationApiVersion: {
+    readonly Latest: "2022-11-01-preview";
+    readonly Stable: "1.0";
+    readonly "1.0": "1.0";
+    readonly "2022-11-01-preview": "2022-11-01-preview";
+};
+
 // @public
 export class AppConfigurationClient {
     constructor(connectionString: string, options?: AppConfigurationClientOptions);
     constructor(endpoint: string, tokenCredential: TokenCredential, options?: AppConfigurationClientOptions);
     addConfigurationSetting(configurationSetting: AddConfigurationSettingParam | AddConfigurationSettingParam<FeatureFlagValue> | AddConfigurationSettingParam<SecretReferenceValue>, options?: AddConfigurationSettingOptions): Promise<AddConfigurationSettingResponse>;
+    archiveSnapshot(snapshotId: SnapshotId, options?: UpdateSnapshotOptions): Promise<UpdateSnapshotResponse>;
+    beginCreateSnapshot(snapshot: SnapshotInfo, options?: CreateSnapshotOptions): Promise<SimplePollerLike<OperationState<CreateSnapshotResponse>, CreateSnapshotResponse>>;
+    beginCreateSnapshotAndWait(snapshot: SnapshotInfo, options?: CreateSnapshotOptions): Promise<CreateSnapshotResponse>;
     deleteConfigurationSetting(id: ConfigurationSettingId, options?: DeleteConfigurationSettingOptions): Promise<DeleteConfigurationSettingResponse>;
     getConfigurationSetting(id: ConfigurationSettingId, options?: GetConfigurationSettingOptions): Promise<GetConfigurationSettingResponse>;
+    getSnapshot(name: string, options?: GetSnapshotOptions): Promise<GetSnapshotResponse>;
     listConfigurationSettings(options?: ListConfigurationSettingsOptions): PagedAsyncIterableIterator<ConfigurationSetting, ListConfigurationSettingPage, PageSettings>;
+    listConfigurationSettingsForSnapshot(snapshotName: string, options?: ListSettingsSnapshotsOptions): PagedAsyncIterableIterator<ConfigurationSetting, ListConfigurationSettingPage, PageSettings>;
     listRevisions(options?: ListRevisionsOptions): PagedAsyncIterableIterator<ConfigurationSetting, ListRevisionsPage, PageSettings>;
+    // Warning: (ae-forgotten-export) The symbol "Snapshot_2" needs to be exported by the entry point index.d.ts
+    listSnapshots(options?: ListSnapshotsOptions): PagedAsyncIterableIterator<Snapshot_2, ListSnapshotsPage, PageSettings>;
+    recoverSnapshot(snapshotId: SnapshotId, options?: UpdateSnapshotOptions): Promise<UpdateSnapshotResponse>;
```

Comment:
The snippets in the README used a method called `addSnapshot` that doesn't appear here.